### PR TITLE
Add release doc and script

### PR DESCRIPTION
**Pull request by arugal**

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,25 +3,34 @@
...
   "main": "index.js",
+  "repository": "apache/skywalking-client-js",
+  "homepage": "skywalking.apache.org",
+  "license": "Apache 2.0",
+  "bugs": {
+    "url": "https://github.com/apache/skywalking/issues",
+    "email": "dev@skywalking.apache.org"
+  },
   "dependencies": {
+    "typescript": "^3.7.2"
```
Comment (kezhenxu94) [inline package.json:14]: Shouldn't this go to the `devDependencies`? It's not runtime dependency

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	npm install
+
+.PHONY: clean
+clean:
+	rm -rf ./node_modules && \
+	rm -rf ./lib
+
+.PHONY: build
+build:
+	npm run build
```
Comment (kezhenxu94) [inline Makefile:54]: Infinite recursive calls to `nom run build` -> `make build`?

Comment (arugal) [reply Makefile:54]: Sorry, it has been fixed :)

File: package.json (hunk 1 above)
Comment (arugal) [reply package.json]: done.

**Review by kezhenxu94 — APPROVED** [review]
LGTM, we now cannot remove a published npm package, be careful, good luck

**Comment by arugal** [comment]
> LGTM, we now cannot remove a published npm package, be careful, good luck

Thx :)

**Pull request merged**